linera-core/src/client/mod.rs

```diff
@@ -610,6 +601,16 @@ impl<P: 'static, S: Storage> ChainClient<P, S> {
         Unsend::new(self.state().inner.map(|state| state.pending_blobs()))
     }
 }
+#[cfg(with_testing)]
```

Comment:
Nit: adding an empty line to separate the items

```suggestion

#[cfg(with_testing)]
```